features/conversation/conversation-chat/conversation-composer/conversation-composer-text-input.tsx

```diff
@@ -47,6 +48,14 @@ export const ConversationComposerTextInput = memo(function ConversationComposerT
     return () => unsubscribe()
   }, [store])
 
+  // Handle prefill value changes
+  useEffect(() => {
+    if (inputDefaultValue && !initialValueRef.current) {
+      initialValueRef.current = inputDefaultValue
+      inputRef.current?.setNativeProps({ text: inputDefaultValue })
+    }
+  }, [inputDefaultValue])
```

Comment:
there's already a store subscription where we listen to the store inputValue and do some changes there, we should use that instead